data/risingwave/models/00_source/marpower/energy.yml

```diff
@@ -0,0 +1,120 @@
+version: 2
+models:
+  - name: battery_dc_converter
+    description: Voltage and current measured by the battery DC converters. {{ doc("marpower_json_explained") }}
+    columns:
+      - name: time
+        description: '{{ doc("zero_timestamp") }}'
+        data_tests:
+            - not_null
+      - name: Voltage_A
+        description: '{{ doc("voltage_at_side") }} A. {{ doc("marpower_json") }}'
+      - name: Voltage_B
+        description: '{{ doc("voltage_at_side") }} B. {{ doc("marpower_json") }}'
+      - name: Current_A
+        description: '{{ doc("current_at_side") }} A. {{ doc("marpower_json") }}'
+      - name: Current_B
+        description: '{{ doc("current_at_side") }} B. {{ doc("marpower_json") }}'
+      - name: topic
+        description: '{{ doc("topic") }}'
+        data_tests:
+            - not_null
+  - name: bms_master
+    description: The stored energy value updates of the batteries. {{ doc("marpower_json_explained") }}
+    columns:
+      - name: time
+        description: '{{ doc("zero_timestamp") }}'
+        data_tests:
+            - not_null
+      - name: Stored_Energy
+        description: '{{ doc("stored_energy") }} {{ doc("marpower_json") }}'
+      - name: topic
+        description: '{{ doc("topic") }}'
+        data_tests:
+            - not_null
+  - name: hydrogeneration
+    description: The electrical energy currently being generated by the thrusters. {{ doc("marpower_json_explained") }}
+    columns:
+      - name: time
+        description: '{{ doc("zero_timestamp") }}'
+        data_tests:
+            - not_null
+      - name: Total_Active_Power
+        description: '{{ doc("total_active_power_prod") }} {{ doc("marpower_json") }}'
+      - name: Total_Power_Factor
+        description: '{{ doc("power_factor") }} {{ doc("marpower_json") }}'
+      - name: topic
+        description: '{{ doc("topic") }}'
+        data_tests:
+            - not_null
+  - name: power_tag
+    description: The electrical energy consumption of devices on the AC grid, measured by the power tags. {{ doc("marpower_json_explained") }}
+    columns:
+      - name: time
+        description: '{{ doc("zero_timestamp") }}'
+        data_tests:
+            - not_null
+      - name: Active_Power_Phase_A
+        description: '{{ doc("active_power_on_phase") }} A. {{ doc("marpower_json") }}'
```

Comment:
For such trivial description I would consider leaving them out. Though perhaps the unit is sensible